parsl/executors/workqueue/executor.py

```diff
@@ -451,15 +124,21 @@ class WorkQueueExecutor(NoStatusHandlingExecutor):
             Queue does not need to transfer and rename files for execution.
             Default is False.
 
+        use_cache: bool
+            Whether workers should cache files that are common to tasks.
+            Warning: Two files are considered the same if they have the same
+            filepath name. Use with care when reusing the executor instance
+            across multiple parsl workflows. Default is False.
+
         source: bool
             Choose whether to transfer parsl app information as
             source code. (Note: this increases throughput for
             @python_apps, but the implementation does not include
             functionality for @bash_apps, and thus source=False
             must be used for programs utilizing @bash_apps.)
-            Default is False.
+            Default is False. Set to True if pack is True
 
-        pack: bool
+         pack: bool
```

Comment:
spuriously introduced space?

Reply:
Fixed in a45233b8775652b5e7f5bf34df2c189a6f22bade